File: Bonrevoir/Bonrevoir/Bonrevoir/ViewController.m
//
//  ViewController.m
//  Bonrevoir
//
//  Created by m2sar on 21/09/2016.
//  Copyright © 2016 UPMC. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    _monLabel.text = @"";
    [_monBouton setTitle:@"Dis Bonjour" forState:UIControlStateNormal];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)actionBouton:(UIButton *)sender {
    if ([_monBouton.titleLabel.text isEqualToString:@"Dis Bonjour"]){
        [_monLabel setText:@"Bonjour"];
        [_monBouton setTitle:@"Dis Aurevoir" forState:UIControlStateNormal];
    }else{
        [_monLabel setText:@"Aurevoir"];
        [_monBouton setTitle:@"Dis Bonjour" forState:UIControlStateNormal];
    }
    
}
@end

File: Bonrevoir/BonrevoirSwift/BonrevoirSwift/ViewController.swift
//
//  ViewController.swift
//  BonrevoirSwift
//
//  Created by m2sar on 22/09/2016.
//  Copyright © 2016 UPMC. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var monLabel: UILabel!

    @IBOutlet weak var buttonoutlet: UIButton!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        monLabel.text = ""
        buttonoutlet.setTitle("Dis Bonjour", for: UIControlState.normal)
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func monBouton(_ sender: UIButton) {
        if(sender.titleLabel?.text == "Dis Aurevoir"){
            monLabel.text = "Aurevoir"
            sender.setTitle("Dis Bonjour", for: UIControlState.normal)
        }else{
            monLabel.text = "Bonjour"
            sender.setTitle("Dis Aurevoir", for: UIControlState.normal)
        }
        
        
        
    }
    

}

